draft-irtf-cfrg-sigma-protocols.md

```diff
@@ -91,7 +91,7 @@ This document describes interactive sigma protocols, a class of secure, general-
 
 # Introduction
 
-Any sigma protocol must define three objects: a *commitment* (computed by the prover), a *challenge* (computed by the verifier), and a *response* (computed by the prover).
+Any sigma protocol must define three objects: a *commitment* (computed by the prover), a *challenge* (computed by the verifier), and a *response* (computed by the prover). One of the advantages of sigma protocols is their composability which enables the construction of more complex protocols. A classic example is the OR composition where the prover seeks to convince the verifier that it knows a valid witness for at least one out of two relations. If there exists a sigma protocol for each relation individually, it is possible to build an OR proof (see Chapter 19.7 of {{BonehS23}} for details). However, such compositions must be handled carefully to preserve security properties as discussed in {{sec-cons}}.
```

Comment:
Perfect. Now just need to add a reference for {{BonehS23}}. For an example, see "SHA3" in the list of informative references higher up in this file.